src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+from uuid import uuid4
+
+from sentry.issues.grouptype import FeedbackGroup
+from sentry.issues.issue_occurrence import IssueEvidence, IssueOccurrence
+from sentry.issues.producer import produce_occurrence_to_kafka
+from sentry.utils.dates import ensure_aware
+
+
+def make_evidence(feedback):
+    evidence_data = {}
+    evidence_display = []
+    if feedback.get("contact_email"):
+        evidence_data["contact_email"] = feedback["contact_email"]
+        evidence_display.append(
+            IssueEvidence(name="contact_email", value=feedback["contact_email"], important=True)
+        )
+    if feedback.get("message"):
+        evidence_data["message"] = feedback["message"]
+        evidence_display.append(
+            IssueEvidence(name="message", value=feedback["message"], important=True)
+        )
+    return evidence_data, evidence_display
+
+
+def create_feedback_issue(event, project_id):
+    event["event_id"] = event.get("event_id") or uuid4().hex
+    evidcence_data, evidence_display = make_evidence(event["feedback"])
+    occurrence = IssueOccurrence(
+        id=uuid4().hex,
+        event_id=event.get("event_id") or uuid4().hex,
+        project_id=project_id,
+        fingerprint=[
+            uuid4().hex
+        ],  # random UUID for fingerprint so feedbacks are grouped individually
+        issue_title="User Feedback",
+        subtitle=event["feedback"]["message"],
```

Comment:
Should this be truncated?

Reply:
Truncation/validation will be handled upstream -- if this ends up throwing here, we can add safeguards.

Reply:
And on subtitle ends up being too long, we can truncate here.